components/tooltip/index.js

```diff
@@ -179,6 +182,7 @@ class Tooltip extends Component {
 					aria-hidden="true"
 				>
 					{ text }
+					{ shortcut && <span className="components-tooltip__shortcut">{ getAccessCombination( shortcut ) }</span> }
```

Comment:
Should the concept of a keyboard shortcut be baked into a general tooltip UI component?

Reply:
I think so, as it has distinct styles. See also #4411.

Reply:
>I think so, as it has distinct styles. See also #4411.

The question was intended as less "Do we need tooltips in the editor to have shortcut-specific behavior?" and more in the abstract "When I use a Tooltip on some page in the admin, do I have the expectation that it supports keyboard shortcut functionality?"

(With the expectation that all of `components` is meant to be purely generic)

In these cases I might like if we could create a wrapper component, similar to the separation between `IconButton` and `Button`.

Reply:
Would it make more sense if the `getAccessCombination` piece is part of `IconButton`, which would otherwise this new wrapper component?

Reply:
I don't see any inherent relationship between an icon button and keyboard combinations, so no, it doesn't really make sense to me.